friends/src/App.js

```diff
@@ -20,10 +17,23 @@ class App extends Component {
     .catch(err => {console.log(err)})
   }
 
+  deleteFriend = (e, id) => {
+    console.log(this.state);
+    axios.delete(`http://localhost:5000/friends/${id}`)
+    .then(res => {
+      console.log(res);
+      this.setState({ friends: res.data })
+    })
+    .catch(err => {
+      console.log(err);
+    })
+  }
+
   render() {
     return (
       <div className="App">
-        {this.state.friends.map(input => <Friend key={input.id} data={input} />)}
+        {this.state.friends.map(friend => <Friend key={friend.id} data={friend} deleteFriend={this.deleteFriend} />)}
+        <FriendForm />
```

Comment:
I'd recommend changing the order here and having your FriendForm render to the page first -- as it is now, the form is at the bottom of a list and keeps getting pushed further down, if a user wants to add consecutive new friends they have to keep scrolling down the page to get at the form